feature-libs/asm/root/services/asm-enabler.service.ts

```diff
@@ -65,6 +81,9 @@ export class AsmEnablerService {
    * Adds the ASM UI by using the `cx-storefront` outlet.
    */
   protected addUi(): void {
-    this.launchDialogService.launch(LAUNCH_CALLER.ASM);
+    this.lazyModule
```

Comment:
We should make sure that the module instance is resolved only once, I'm thinking about wrapping it into observable, for example `asmModuleInstance$` with `shareReplay`